plugins/inventory/vmware_vm_inventory.py

```diff
@@ -69,67 +69,69 @@
             description:
             - Specify the list of VMware schema properties associated with the VM.
             - These properties will be populated in hostvars of the given VM.
-            - Each value in the list specifies the path to a specific property in VM object.
+            - Each value in the list can be a path to a specific property in VM object or a path to a collection of VM objects.
+            - Set value to 'all' to query all properties.
```

Comment:
This instruction isn't entirely clear. The param type is a list, so what does it mean to set value of `'all'`? If you told me to use `['all']`, then I would be all good 👌

Reply:
list types allow for 'single element' and will be automatically wrapped, but still we should encourage the 'correct' input

Reply:
Your are right. I was think about readable. However, I will ensure that `[ALL]` is acceptable input too.

Reply:
Can you please review new changes?
@AlanCoding